lib/modules/datasource/rubygems/metadata-cache.spec.ts

```diff
@@ -8,17 +8,18 @@ jest.mock('../../../util/cache/package');
 const packageCache = mocked(_packageCache);
 
 describe('modules/datasource/rubygems/metadata-cache', () => {
-  const cache: Map<string, unknown> = new Map();
+  const packageCacheMock: Map<string, unknown> = new Map();
 
   beforeEach(() => {
-    cache.clear();
+    packageCacheMock.clear();
 
     packageCache.get.mockImplementation(
-      (ns, key) => Promise.resolve(cache.get(`${ns}::${key}`)) as never
+      (ns, key) =>
+        Promise.resolve(packageCacheMock.get(`${ns}::${key}`)) as never
```

Comment:
do we really need the never cast?

Reply:
Either we cast to never or declare value type as `any`